source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: "Home Assistant Companion for Mac"
+description: "Integrate your Mac into Home Assistant."
+date: 2020-09-18 00:00:00
+date_formatted: "September 18, 2020"
+comments: true
+author: Tom Brien
+author_twitter: tbrien88
+categories: Announcements
+og_image: /images/blog/2020-09-15-home-assistant-tags/social.png
```

Comment:
Social image needs updating.

Reply:
Resolving this one as I have now changed the meta to point to the right file... I will see what it looks like with a windowed view but might keep that image full screened as the OG image can be pretty small in places